Initial D8 port

kimpepper opened the pull request.

rcaracaus commented:
I think initial commits like this need to be reviewed as well with some steps to test and a short explanation of what you did.

kimpepper commented:
I deleted all custom and contrib modules, and updated composer to be able
to run drush install. It's pretty much an empty D8 site with a shell of an
install profile.

On 18 August 2015 at 15:01, Robert Caracaus <notifications@github.com>
wrote:

> I think initial commits like this need to be reviewed as well with some
> steps to test and a short explanation of what you did.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/previousnext/agov/pull/634#issuecomment-132072126>.
>



-- 
Kim Pepper
Technical Director
PreviousNext

desk: 02 8005 8670
mobile: 0414 212 530
skype: kim.pepper
